src/main/java/org/apache/commons/compress/utils/TimeUtils.java

```diff
@@ -46,6 +46,62 @@ public final class TimeUtils {
      */
     static final long WINDOWS_EPOCH_OFFSET = -116444736000000000L;
 
+    /**
+     * Converts "standard UNIX time" (in seconds, UTC/GMT) to {@link FileTime}.
+     *
+     * @param time UNIX timestamp
+     * @return the corresponding FileTime
+     */
+    public static FileTime unixTimeToFileTime(final long time) {
+        return FileTime.from(time, TimeUnit.SECONDS);
+    }
+
+    /**
+     * Converts {@link FileTime} to "standard UNIX time".
+     *
+     * @param time the original FileTime
+     * @return the UNIX timestamp
+     */
+    public static long fileTimeToUnixTime(final FileTime time) {
```

Comment:
The method prefix "fileTime" is redundant with the argument type, `fileTimeToUnixTime` -> `toUnixTime`. Since Unix time is always in seconds (as a primitive long here), we should not need to include the scale in the name.

Reply:
The scale isn't included in the name, is it?